src/templates/podcast.js

```diff
@@ -83,6 +80,29 @@ const StyledPlayer = styled(AudioPlayer)`
     }
 `;
 
+const StyledBlogShareLinks = styled(BlogShareLinks)`
+    flex-direction: column;
+    align-items: center;
+    > :not(:first-child) {
+        margin-top: ${size.medium};
+    }
+    @media ${screenSize.upToLarge} {
+        display: inline-flex;
+        flex-direction: row;
+        > * {
+            margin-top: 0;
+            margin-left: ${size.mediumLarge};
```

Comment:
<img width="236" alt="Screenshot 2021-05-17 at 23 53 54" src="https://user-images.githubusercontent.com/79590294/118555328-3c8a1c80-b76b-11eb-834d-99571c50f296.png">
Looks like we don't need `margin-left` for the first element, let's omit first element.